### PR TITLE
Add AS923 frequency plan

### LINKED ISSUE #16: AS923 Frequency Plan
#### Summary

We may need a basic AS923 frequency plan (with only the mandatory channels).

#### Why do we need this?

For compatibility with v2, which uses such a frequency plan if it can't reliably determine the country of the gateway.

#### How do you propose to implement this?

Copy-paste from another AS923 plan, remove channels and change the min/max.

### PR DESCRIPTION
**Pull request by KrishnaIyer**
#### Summary

Closes #16 

#### Changes

- Remove the extra channels from `AS_920_923`
- Checked with this; https://github.com/TheThingsNetwork/gateway-conf/blob/master/AS1-global_conf.json for the other fields.

I've chosen the min-max for `radio0` from LW Regional Parameters 1.0.3 RevA
image

#### Notes for Reviewers

I didn't find any concrete documentation for the `radios`/`fsk`/`lora-std` configs so I can't really say if this is correct or not.

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible, they don't break existing deployments.
- [ ] Testing: The changes are tested.
- [ ] Documentation: Relevant documentation is added or updated.

### CODE REVIEW

File: AS_923.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  min-data-rate: 0
+  max-data-rate: 5
+lora-standard-channel:
+  frequency: 922100000
+  data-rate: 6
+  radio: 1
+fsk-channel:
+  frequency: 921800000
+  data-rate: 7
+  radio: 1
```
Comment (htdvisser) [inline AS_923.yml:25]: The high-bandwidth and FSK channels don't belong in here. Only the two mandatory channels.

File: AS_923.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  chip-type: SX1257
+  frequency: 923000000
+  rssi-offset: -166
+  tx:
+    min-frequency: 915000000
+    max-frequency: 928000000
+- enable: true
+  chip-type: SX1257
+  frequency: 922000000
+  rssi-offset: -166
```
Comment (htdvisser) [inline AS_923.yml:37]: I think you can disable the second radio, since it isn't being used after deleting the high-bandwidth and FSK channels.

**Comment by KrishnaIyer** [comment]
Waiting on your review before merging @laurensslats

**Comment by johanstokking** [comment]
Let's merge, this description is good

**Pull request merged**